core/src/utils.jl

```diff
@@ -12,25 +12,97 @@ function valid_edge_types(db::DB)::Bool
     return !errors
 end
 
-"Return a directed graph, and a mapping from source and target nodes to edge fid."
-function create_graph(
-    db::DB,
-    edge_type_::String,
-)::Tuple{DiGraph, Dictionary{Tuple{Int, Int}, Int}, Dictionary{Int, Tuple{Symbol, Symbol}}}
+"""
+Return a directed metagraph with data of nodes (NodeMetadata):
+- Node type (snake case)
+- Allocation network ID
+
+and data of edges (EdgeMetadata):
+- Edge ID (EdgeID)
+- type (flow/control)
```

Comment:
The content of NodeMetadata and EdgeMetadata are also in those docstrings. Rather than copying, it's better to link to those, using

```
[`NodeMetadata`](@ref)
```